### PR TITLE
Display teams

### PR DESCRIPTION
**Pull request by GabrielAdward**

### CODE REVIEW

File: football-manager/src/lib/Pages/DisplayTeams/DTNavbar.svelte [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+  </script>
+  
+  <div class="bg-slate-400 dark:bg-slate-800 w-full rounded-xl p-2 flex gap-2">
+    <NavButton buttonText="Home" location="/club/{id}" {id} />
+    <NavButton buttonText="Teams" location="/club/{id}/teams" {id} />
+    <NavButton buttonText="Calendar" location="/team/{id}/calendar" {id} />
+    <NavButton buttonText="Contact" location="/team/{id}/contact" {id} />
+    <!-- Only visible to admins: -->
+    <NavButton buttonText="Admin" location="/team/{id}/admin" {id} />
+  </div>
```
Comment (albinforsbergju) [inline football-manager/src/lib/Pages/DisplayTeams/DTNavbar.svelte:14]: No need for this, we already have a navbar for club.

File: football-manager/src/lib/Pages/DisplayTeams/Teams.svelte [hunk 2]
```diff
@@ -0,0 +1,10 @@
+<script>
+  export let id;
+  import DtNavbar from "./DTNavbar.svelte";
+  import DisplayAllTeams from "./DisplayAllTeams.svelte";
+
+</script>
+
+<DtNavbar {id} />
```
Comment (albinforsbergju) [inline football-manager/src/lib/Pages/DisplayTeams/Teams.svelte:8]: Use club navbar

**Review by albinforsbergju — CHANGES_REQUESTED** [review]

File: football-manager/src/lib/Pages/DisplayTeams/DisplayAllTeams.svelte [hunk 3]
```diff
@@ -0,0 +1,31 @@
+<script>
+      import teamsData from "../../../assets/TeamsData";
+</script>
+
+<section class="animate-fade-in flex-col mt-10">
```
[the remark this answers is not in the document]
Comment (GabrielAdward) [reply football-manager/src/lib/Pages/DisplayTeams/DisplayAllTeams.svelte:5]: Talked about it in school, will see how we set up the database and then change what needs to be changed.

File: football-manager/src/App.svelte [hunk 4]
```diff
@@ -19,8 +20,12 @@
   <Route path="/club/:id" let:params>
     <Club id="{params.id}" />
   </Route>
+  <Route path="/club/:id/teams" let:params> 
+    <Teams id="{params.id}"/>
+
```
Comment (albinforsbergju) [inline football-manager/src/App.svelte:25]: Route element not closed. App won't start.

**Review by albinforsbergju — CHANGES_REQUESTED** [review]
Fix!

**Review by albinforsbergju — APPROVED** [review]
LGTM!

**Pull request merged**